xml/System/MidpointRounding.xml

```diff
@@ -187,7 +187,7 @@
       </ReturnValue>
       <MemberValue>3</MemberValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>The result closest to and no greater than the infinitely precise result.</summary>
```

Comment:
These should likely be prefixed with `When a number is halfway between two others, it is rounded toward`

Reply:
Like this?:

```suggestion
        <summary>When a number is halfway between two others, it is rounded toward
 the result closest to and no greater than the infinitely precise result.</summary>
```